src/parser/jobs/mnk/modules/DragonKick.tsx

```diff
@@ -0,0 +1,67 @@
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {Cause, Event} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {filter} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const SUGGESTION_TIERS = { // DK is 320, Bootshine is 210 without the buff, so you effective lost 110 * critcial hit modifers,  so it adds up fast.
+	1: SEVERITY.MINOR, // minor loss
+	2: SEVERITY.MEDIUM, // You've lost a full Bootshine and then some
+	3: SEVERITY.MAJOR, // You've got a problem.
+	// 4: SEVERITY.JUSTWHY Dear God man, use your hands!
+}
+
+export class DragonKick extends Analyser {
+	static override handle = 'dragonkick'
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	private LeadBuffUp = false
+	private overKicks = 0 // Amount of times they kicked over a buff
+
+	override initialise(): void {
+		this.addEventHook(
+			filter<Event>()
+				.source(this.parser.actor.id)
+				.type('damage')
+				.cause(filter<Cause>().action(this.data.actions.DRAGON_KICK.id)),
```

Comment:
Not sure how I feel about this. It's better this way if you're manually tracking their status state, but if going for the more basic "do I have this status" you need to check on use time instead of damage time, unless it misses. Kinda annoying now I think about it more. An observation tho.